source/features/clean-dashboard.css

```diff
@@ -8,8 +8,8 @@
 */
 
 /* Hide duplicate repo name in some non-grouped events */
-.rgh-clean-dashboard .dashboard :is(.watch_started, .repo, .public) :is(.color-border-secondary, .color-border-muted).flex-items-baseline .text-bold:is(.color-text-primary, .color-fg-default) {
-	display: none;
+.rgh-clean-dashboard .dashboard :is(.watch_started, .repo, .public) :is(.color-border-secondary, .color-border-muted).flex-items-baseline [data-hovercard-type='user'] ~ .Link--primary {
+	display: none !important;
 }
```

Comment:
Previously this hides the name inside the card, making it look weird and inconsistent with other cards. This hides the link _outside_ the card, next to the username instead.